auth-tokens/src/test/java/com/palantir/tokens/auth/AuthHeaderTest.java

```diff
@@ -32,6 +32,31 @@ public void testSimple() {
         assertThat(AuthHeader.valueOf(authHeader.toString()), is(authHeader));
     }
 
+    @Test(expected = IllegalArgumentException.class)
+    public void testMissingToken() {
+        AuthHeader.valueOf("Bearer");
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testMissingBearer() {
+        AuthHeader.valueOf("someToken");
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testMultipleTokens() {
+        AuthHeader.valueOf("Bearer tokenOne tokenTwo");
```

Comment:
it seems like this fails due to being a bad token format, not because it doesn't match the regex